app/src/main/java/com/sourcepoint/test_project/MainActivity.java

```diff
@@ -36,6 +36,7 @@ private void removeView(View view) {
     private GDPRConsentLib buildGDPRConsentLib() {
         return GDPRConsentLib.newBuilder(22, "mobile.demo", 2372,"5c0e81b7d74b3c30c6852301",this)
                 .setStagingCampaign(false)
+                .setAuthId("branco005")
```

Comment:
Leo I wouldn't keep the authId in the example app, maybe you forgot to remove it?

Reply:
yep, totally forgot it there 🤦‍♂